ospf6d/ospf6_neighbor.c

```diff
@@ -981,6 +995,14 @@ static void ospf6_neighbor_show_detail(struct vty *vty,
 			vty_out(vty, "      %s\n", lsa->name);
 
 		bfd_sess_show(vty, NULL, on->bfd_session);
+
+		if (on->auth_present == true) {
+			vty_out(vty, "    Authentication is enabled\n");
+			vty_out(vty, "      Higher sequence no %d, ",
+				on->seqnum_h);
+			vty_out(vty, "Lower sequence no %d\n", on->seqnum_l);
+		} else
+			vty_out(vty, "    Authentication is disabled\n");
```

Comment:
(same here)

Reply:
Yes. It means that Authentication is disabled in neighbor. This display it to show that auth trailer is disabled in ospf6 neighbor.